src/components/navBar/index.tsx

```diff
@@ -11,31 +11,73 @@ import {
   MEMBERS_URL,
   STATUS_URL,
   GITHUB_LOGO,
-  RDS_LOGO
-} from '@/components/constants/url'
-import Dropdown from '../Dropdown/Dropdown';
-import styles from '@/components/navBar/navBar.module.scss';
+  RDS_LOGO,
+  CROSS,
+} from "@/components/constants/url";
+import Dropdown from "../Dropdown/Dropdown";
+import styles from "@/components/navBar/navBar.module.scss";
 
 const NavBar = () => {
   const { userData, isLoggedIn } = useAuthenticated();
   const [toggleDropdown, setToggleDropdown] = useState(false);
+  const [toggleDrawer, setToggleDrawer] = useState(false);
```

Comment:
cant both of them rest in the same state? or is there a reason?

Reply:
No, they both maintain two separate states, hence using only one state for both will trigger dropdown as well as the drawer at the same time.